torch/_inductor/codegen/cpp.py

```diff
@@ -2083,6 +2095,7 @@ def get_reduction_code_buffer(loops, buffer="prefix"):
                         if buffer == "local":
                             return (
                                 kernel.local_reduction_init,
+                                kernel.parallel_reduction_weight_recps,
```

Comment:
why can't we reuse the `local_reduction_init` here?

Reply:
We can reuse `local_reduction_init `. Modified